README.md

```diff
@@ -232,7 +232,7 @@ Table of Contents
   * https://www.incapsula.com/ - free CDN and DDoS protection
 
 ## PaaS
-
+  * http://restlet.com/products/apispark/ - APISpark enables any API, application or data owner to become an API provider in minutes via an intuitive browser interface.
```

Comment:
It's closer to IaaS than PaaS, regardless their overreaching marketing collateral.  You just have to look at the feature lists of others in the category to know this one is not a PaaS.